packages/addons/addon-depends/moonlight-embedded/patches/01-flush-code.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/src/main.c b/src/main.c
```

Comment:
This is about all patches for moonlight-embedded. It seems you are the maintainer. Why are patches required? Isn't it better to incorporate it in the git repo? Maybe you have your reasons.

Reply:
This patch should definitely be upstreamed.

Reply:
If you're willing to push this commit to upstream, I'd be happy to remove this one.

These two patches I considered as 'hacky' code, not intended to be in a main branch, as it's supposed to be replaced by code that will work in any environment.